book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+
+  attr_accessor :title, :author
+
+  # has-many relation with Rental
+  def rentals
+    Rental.all.select { |rental| rental.book == self }
+  end
```

Comment:
- [ ] dear @Luffytaro22, inside the book class, we need to initialize the rentals, as `@rentals = []`. Also once you add the rentals to the initializer please create `attr_accessor` to it. We need to be able to check the rentals for the book. 
- [ ] Also, when we add a rental we need to create a new rental with the book as a self and add the newly created rental to the rentals list, This way the association will be preserved all the way. 

> The method should look something like...
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```